storage/innobase/xtrabackup/utils/percona-xtrabackup-8.0_builder.sh

```diff
@@ -0,0 +1,587 @@
+    PXBDIR=${EXPORTED_TAR%.tar.gz}
+    rm -rf ${PXBDIR}
+    tar xzf ${EXPORTED_TAR}
+    rm -f ${EXPORTED_TAR}
+
+    cd ${PXBDIR}
+    sed -i "s:@@XB_VERSION_MAJOR@@:${XB_VERSION_MAJOR}:g" storage/innobase/xtrabackup/utils/percona-xtrabackup.spec
+    sed -i "s:@@XB_VERSION_MINOR@@:${XB_VERSION_MINOR}:g" storage/innobase/xtrabackup/utils/percona-xtrabackup.spec
+    sed -i "s:@@XB_VERSION_PATCH@@:${XB_VERSION_PATCH}:g" storage/innobase/xtrabackup/utils/percona-xtrabackup.spec
+    #
+    if [ -z "${XB_VERSION_EXTRA}" ]; then
+    EXTRAVER="%{nil}"
+    #  RPM_EXTRAVER="%{nil}"
+    RPM_EXTRAVER=${RPM_RELEASE}
+    else 
+    EXTRAVER=${XB_VERSION_EXTRA}
+    #  RPM_EXTRAVER=.${XB_VERSION_EXTRA#-}
+    RPM_EXTRAVER=${XB_VERSION_EXTRA#-}.${RPM_RELEASE}
+    fi
+    #
+    sed -i "s:@@XB_VERSION_EXTRA@@:${EXTRAVER}:g" storage/innobase/xtrabackup/utils/percona-xtrabackup.spec
+    sed -i "s:@@XB_RPM_VERSION_EXTRA@@:${RPM_EXTRAVER}:g" storage/innobase/xtrabackup/utils/percona-xtrabackup.spec
+    sed -i "s:@@XB_REVISION@@:${REVISION}:g" storage/innobase/xtrabackup/utils/percona-xtrabackup.spec
+    #
+    #for 8.0 alpha release
+    sed -i "s:= %{version}-%{release}:> 8.0:g" storage/innobase/xtrabackup/utils/percona-xtrabackup.spec
+
+    # create a PXB tar
+    cd ${WORKDIR}/percona-xtrabackup
+    tar --owner=0 --group=0 --exclude=.bzr --exclude=.git -czf ${PXBDIR}.tar.gz ${PXBDIR}
+    rm -rf ${PXBDIR}
+
+    mkdir $WORKDIR/source_tarball
+    mkdir $CURDIR/source_tarball
+    cp ${PXBDIR}.tar.gz $WORKDIR/source_tarball
+    cp ${PXBDIR}.tar.gz $CURDIR/source_tarball
+    cd $CURDIR
+    rm -rf percona-xtrabackup
+    return
+}
+
+get_system(){
+    if [ -f /etc/redhat-release ]; then
+        RHEL=$(rpm --eval %rhel)
+        ARCH=$(echo $(uname -m) | sed -e 's:i686:i386:g')
+        OS_NAME="el$RHEL"
+        OS="rpm"
+    else
+        ARCH=$(uname -m)
+        OS_NAME="$(lsb_release -sc)"
+        OS="deb"
+    fi
+    return
+}
+
+install_deps() {
+    if [ $INSTALL = 0 ]
+    then
+        echo "Dependencies will not be installed"
+        return;
+    fi
+    if [ ! $( id -u ) -eq 0 ]
+    then
+        echo "It is not possible to instal dependencies. Please run as root"
+        exit 1
+    fi
+    CURPLACE=$(pwd)
+    if [ "x$OS" = "xrpm" ]
+    then
+        add_percona_yum_repo
+        yum -y install git wget 
+        if [[ "${RHEL}" -eq 8 ]]; then         
+            PKGLIST+=" binutils-devel python3-pip python3-setuptools python3-wheel"
+            PKGLIST+=" libcurl-devel cmake libaio-devel zlib-devel libev-devel bison make gcc"
+            PKGLIST+=" rpm-build libgcrypt-devel ncurses-devel readline-devel openssl-devel gcc-c++"
+            PKGLIST+=" install vim-common rpmlint"
+            until yum -y install ${PKGLIST}; do
+                echo "waiting"
+                sleep 1
+            done
+            /usr/bin/pip3 install sphinx
+        else
+            until yum -y install epel-release centos-release-scl; do
```

Comment:
epel-release is available on all centos system. so we can  install it everywhere

Reply:
It is. But there are no EPEL packages installed for Cent8, so I decided to keep it for >8 versions